lib/backend/etcdbk/etcd.go

```diff
@@ -217,9 +218,25 @@ func New(ctx context.Context, params backend.Params) (*EtcdBackend, error) {
 		watchDone:        make(chan struct{}),
 		buf:              buf,
 	}
+
 	if err = b.reconnect(); err != nil {
 		return nil, trace.Wrap(err)
 	}
+
+	// Check that the etcd nodes are at least the minimum version supported
+	timeout, cancel := context.WithTimeout(ctx, time.Second)
```

Comment:
This timeout is too tight, especially with checking versions sequentially.
Make it something like `len(cfg.Nodes) * 3 * time.Second`